epplib/models/info.py

```diff
@@ -153,7 +152,7 @@ def _get_params(cls, element: Element) -> Mapping[str, Any]:
         contacts = cls._find_all(element, f"./{cls._namespace}:contact")
         if ns is not None:
```

Comment:
In this context I don't believe so, since its running `cls._find_all_text(ns, f"./{cls._namespace}:hostObj")` instead of `[DomainContact.extract(item) for item in contacts]`